sdk/communication/azure-communication-jobrouter/azure/communication/jobrouter/_router_client.py

```diff
@@ -0,0 +1,1219 @@
+            worker_id = worker_id,
+            **kwargs
+        )
+    # endregion Worker
+
+    # region Job
+
+    @distributed_trace
+    def create_job(
+            self,
+            job_id: str,
+            channel_id: str,
+            *,
+            channel_reference: Optional[str] = None,
+            classification_policy_id: Optional[str] = None,
+            queue_id: Optional[str] = None,
+            priority: Optional[int] = None,
+            requested_worker_selectors: Optional[List[WorkerSelector]] = None,
+            labels: Optional[Dict[str, Union[int, float, str, bool]]] = None,
+            tags: Optional[Dict[str, Union[int, float, str, bool]]] = None,
+            notes: Optional[Dict[datetime, str]] = None,
+            **kwargs: Any
+    ) -> RouterJob:
+        """ Create a job.
+
+        :param str job_id: Id of the job.
+
+        :param str channel_id: The channel identifier. eg. voice, chat, etc.
+
+        :keyword channel_reference: Reference to an external parent context, eg. call ID.
+        :paramtype channel_reference: Optional[str]
+
+        :keyword classification_policy_id: The Id of the Classification policy used for classifying a
+         job.
+        :paramtype classification_policy_id: Optional[str]
+
+        :keyword queue_id: The Id of the Queue that this job is queued to.
+        :paramtype queue_id: Optional[str]
+
+        :keyword priority: The priority of this job.
+        :paramtype priority: Optional[int]
+
+        :keyword requested_worker_selectors: A collection of manually specified label selectors, which
+         a worker must satisfy in order to process this job.
+        :paramtype requested_worker_selectors: Optional[List[~azure.communication.jobrouter.WorkerSelector]]
+
+        :keyword labels: A set of key/value pairs that are identifying attributes used by the rules
+         engines to make decisions.
+        :paramtype labels: Optional[Dict[str, Union[int, float, str, bool]]]
+
+        :keyword tags: A set of tags. A set of non-identifying attributes attached to this job.
+        :paramtype tags: Optional[Dict[str, Union[int, float, str, bool]]]
+
+        :keyword notes: Notes attached to a job, sorted by timestamp.
+        :paramtype notes: Optional[Dict[~datetime.datetime, str]]
+
+
+        :return: RouterJob
+        :rtype: ~azure.communication.jobrouter.RouterJob
+        :raises: ~azure.core.exceptions.HttpResponseError, ValueError
+
+        .. admonition:: Example:
+
+            .. literalinclude:: ../samples/router_job_crud_ops.py
+                :start-after: [START create_job]
+                :end-before: [END create_job]
+                :language: python
+                :dedent: 8
+                :caption: Use a RouterClient to create a job
+        """
+        if not job_id:
+            raise ValueError("job_id cannot be None.")
+
+        if not channel_id:
+            raise ValueError("channel_id cannot be None")
```

Comment:
We shouldn't validate body parameters. The service will do it.

Reply:
removed